src/tower/__main__.py

```diff
@@ -47,7 +47,13 @@ def on_draw(self):
 
         self.scene.draw()
 
+    def get_asset_path(self, file_name: str) -> str:
```

Comment:
I don't think this should be a method, should just be a function if we're going this route.

Reply:
+1, it should just be a utility function outside of the class

Reply:
Will change it, took a 50/50 guess.